src/main/java/io/debezium/connector/yugabytedb/YugabyteDBStreamingChangeEventSource.java

```diff
@@ -940,7 +940,7 @@ public void commitOffset(Map<String, ?> offset) {
                     // than one already present would throw the error: CDCSDK: Trying to fetch already GCed intents
                     if (this.tabletToExplicitCheckpoint.get(entry.getKey()) != null &&
                             tempOpId.getIndex() < this.tabletToExplicitCheckpoint.get(entry.getKey()).getIndex()) {
-                        LOGGER.warn("The received OpId {} is less than the older checkpoint {} for tablet {}",
+                        LOGGER.debug("The received OpId {} is less than the older checkpoint {} for tablet {}",
```

Comment:
This is converted to `debug` level considering the following scenario:
1. Last record published by the connector has OpId `1.5`
2. Connector received a callback on the above OpId so explicit checkpoint is also at `1.5`
3. Now there were couple of NO_OP on service so connector received empty batches and explicit checkpoint was advanced to `1.8`
4. Assume that for sometime there are no records, or there are no records after the record in step 1
5. This will result in a commit callback with checkpoint `1.5` since that was the last published record
6. Subsequently, we will always end up printing this warning log which could be confusing to users.